styles/index.css

```diff
@@ -52,3 +116,83 @@
   font-family: var(--heading-sans);
   font-weight: 600;
 }
+.left-corner p {
+  margin-left: 0;
+}
+.blogs-section-container {
+  padding: 30px 88px;
+  font-family: var(--paragraph-sans);
+}
+.blogs-header-container {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+.blogs-header-container h1,
+p {
+  margin: 10px;
+}
+.blogs-header-container h1 {
+  padding: 0;
+}
+.blogs-carousel {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.left-button,
+.right-button {
+  width: 15px;
+  height: 15px;
+  border: none;
+  background-color: inherit;
+  cursor: pointer;
+  padding: 0;
+  margin: 0;
+}
+.carousel-buttons {
```

Comment:
The whole segment appears to be a carousel itself. Do you consider re-organize their naming

Reply:
Yes, I renamed the container to be blogs-carousel with "right" and "left-button"